tests/timetables/test_interval_timetable.py

```diff
@@ -35,11 +35,32 @@
 PREV_DATA_INTERVAL = DataInterval(start=PREV_DATA_INTERVAL_START, end=PREV_DATA_INTERVAL_END)
 
 CURRENT_TIME = pendulum.DateTime(2021, 9, 7, tzinfo=TIMEZONE)
+YESTERDAY = CURRENT_TIME - datetime.timedelta(days=1)
 
 HOURLY_CRON_TIMETABLE = CronDataIntervalTimetable("@hourly", TIMEZONE)
 HOURLY_TIMEDELTA_TIMETABLE = DeltaDataIntervalTimetable(datetime.timedelta(hours=1))
 HOURLY_RELATIVEDELTA_TIMETABLE = DeltaDataIntervalTimetable(dateutil.relativedelta.relativedelta(hours=1))
 
+CRON_TIMETABLE = CronDataIntervalTimetable("30 16 * * *", TIMEZONE)
+CRON_INTERVAL = datetime.timedelta(minutes=30, hours=16)
```

Comment:
This `CRON_INTERVAL` name is confusing; the “interval” of the timetable is actually 24 hours (because jobs run everyday at 16:30). Let’s change it (although I’m not sure what name is appropriate).

Reply:
Yep, you are right. How about using INTERVAL_FROM_ZERO？:smiley:

Reply:
Maybe `DELTA_FROM_MIDNIGHT` (zero sounds wrong)?

Reply:
Oh, Great!  Use it